examples/evm-to-btc-fungible-transfer/.env.sample

```diff
@@ -0,0 +1,4 @@
+SEPOLIA_RPC_URL=
+BTC_DESTINATION_ADDRESS=
+PRIVATE_KEY=
```

Comment:
probably we can add at least dummy data as an example

Reply:
dummy data like this?

```
PRIVATE_KEY=""
CRONOS_RPC_URL="CRONOS_RPC_URL_HERE"
SEPOLIA_RPC_URL="SEPOLIA_RPC_URL_HERE"
SYGMA_ENV="testnet"
```

Reply:
@wainola yeah, something like this

Reply:
ok